docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -47,21 +51,27 @@ A DaemonSet also needs a [`.spec`](https://git.k8s.io/community/contributors/dev
 
 ### Pod Template
 
-The `.spec.template` is the only required field of the `.spec`.
+The `.spec.template` is one of the required fields `.spec`.
 
-The `.spec.template` is a [pod template](/docs/concepts/workloads/pods/pod-overview/#pod-templates). It has exactly the same schema as a [pod](/docs/concepts/workloads/pods/pod/), except it is nested and does not have an `apiVersion` or `kind`.
+The `.spec.template` is a [Pod Template](/docs/concepts/workloads/pods/pod-overview/#pod-templates). It has exactly the same schema as a [Pod](/docs/concepts/workloads/pods/pod/), except it is nested and does not have an `apiVersion` or `kind`.
 
-In addition to required fields for a pod, a pod template in a DaemonSet has to specify appropriate
-labels (see [pod selector](#pod-selector)).
+In addition to required fields for a Pod, a Pod template in a DaemonSet has to specify appropriate
+labels (see [Pod Selector](#pod-selector)).
 
-A pod template in a DaemonSet must have a [`RestartPolicy`](/docs/user-guide/pod-states)
+A Pod Template in a DaemonSet must have a [`RestartPolicy`](/docs/user-guide/pod-states)
  equal to `Always`, or be unspecified, which defaults to `Always`.
 
 ### Pod Selector
 
-The `.spec.selector` field is a pod selector.  It works the same as the `.spec.selector` of
+The `.spec.selector` field is a Pod Selector.  It works the same as the `.spec.selector` of
 a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) or other new resources.
 
```

Comment:
If you're going to talk generically about selecting Pods, you would use Pod selector (lower-case s).

If you're going to talk about API objects, then use caps, but use the actual API object like this:
The spec.selector field is a LabelSelector.